src/commands/postDeploy/checkLinuxWebAppDownDetector.ts

```diff
@@ -58,18 +57,32 @@ export async function checkLinuxWebAppDownDetector(correlationId: string, node:
                 return undefined;
             }
 
-            // tslint:disable-next-line: no-unsafe-any
-            detectorErrorMessage = await getLinuxDetectorError(context, linuxContainerStartFailureId, node, deployResultTime, node.root.client.fullName);
+            // time constraint of being less than the current time by 14.5 minutes is enforced system wide for all detectors
+            // however LinuxLogViewer detector ignores it and gets the most recent data
+            const nowTime: Date = new Date();
+            const startTimeDate: Date = new Date(nowTime.getTime() - (60 * 60 * 1000));
+            const endTimeDate: Date = new Date(nowTime.getTime() - (30 * 60 * 1000));
+
+            // need to remove the miliseconds from the ISOstring for request
+            const startTimeArray: string[] = startTimeDate.toISOString().split(':');
+            const startTime: string = `${startTimeArray[0]}:${startTimeArray[1]}`;
+
+            const endTimeArray: string[] = endTimeDate.toISOString().split(':');
+            const endTime: string = `${endTimeArray[0]}:${endTimeArray[1]}`;
```

Comment:
How about you use `moment` to format exactly the date you want instead of doing this splitting stuff

Same thing as here: https://github.com/microsoft/vscode-azuretools/pull/658/files#diff-e1557ab1c14775da7a5262caf71fee73R22